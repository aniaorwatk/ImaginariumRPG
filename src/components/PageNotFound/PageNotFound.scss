@import "./../scss/_Main.scss";

.pageNotFound {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    text-align: center;
    height: 100vh;
    width: 100vw;
    font-size: 2rem;
    color: $color-white;
}