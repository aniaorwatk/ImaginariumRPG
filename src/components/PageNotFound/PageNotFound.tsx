import labels from "../../labels"
import "./PageNotFound.css"

const PageNotFound = () => {

    return (
        <div className="pageNotFound">
            <p>{labels.pageNotFound.infAboutError}</p>
            <p>{labels.pageNotFound.pageNotFound}</p>
        </div>
    )
}

export default PageNotFound
