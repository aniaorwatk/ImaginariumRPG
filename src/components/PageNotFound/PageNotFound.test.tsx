import { render } from "@testing-library/react";
import PageNotFound from "./PageNotFound";

describe('PageNotFound', () => {
    it('Should render', () => {
        const { container } = render(<PageNotFound />);
        expect(container).toMatchSnapshot();
    })
})
