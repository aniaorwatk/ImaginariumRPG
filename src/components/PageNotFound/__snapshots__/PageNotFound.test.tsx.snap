// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound Should render 1`] = `
<div>
  <div
    class="pageNotFound"
  >
    <p>
      Ups!!! Coś poszło nie tak.
    </p>
    <p>
      Strona nie odpowiada.
    </p>
  </div>
</div>
`;
